misc/task-server-endpoint-validator/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:latest
```

Comment:
Locking this to a version would be a good idea. We are using 16.04 at other places.

Reply:
make sense :)